media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
+  encoding: ASCII
+doc: |
+  The WAVE file format is a subset of Microsoft's RIFF specification for the
+  storage of multimedia files. A RIFF file starts out with a file header
+  followed by a sequence of data chunks. A WAVE file is often just a RIFF
+  file with a single "WAVE" chunk which consists of two sub-chunks --
+  a "fmt " chunk specifying the data format and a "data" chunk containing
+  the actual sample data.
+  This Kaitai implementation was written by John Byrd of Gigantic Software
+  (jbyrd@giganticsoftware.com), and it is likely to contain bugs.
+doc-ref: http://soundfile.sapp.org/doc/WaveFormat/
+doc-ref: https://www.loc.gov/preservation/digital/formats/fdd/fdd000001.shtml
```

Comment:
I'd actually go with a link to "official" (i.e. Microsoft's documentation) in this case, i.e. http://www-mmsp.ece.mcgill.ca/Documents/AudioFormats/WAVE/Docs/RIFFNEW.pdf

`loc.gov/.../fdd/...` link should be probably in `meta/xref` in abbreviated form.